pb/profile.pb.go

```diff
@@ -121,50 +131,82 @@ func (m *Profile_Image) String() string            { return proto.CompactTextStr
 func (*Profile_Image) ProtoMessage()               {}
 func (*Profile_Image) Descriptor() ([]byte, []int) { return fileDescriptor3, []int{0, 1} }
 
+type Profile_Stats struct {
+	FollowerCount  uint32 `protobuf:"varint,1,opt,name=followerCount" json:"followerCount,omitempty"`
+	FollowingCount uint32 `protobuf:"varint,2,opt,name=followingCount" json:"followingCount,omitempty"`
+	ListingCount   uint32 `protobuf:"varint,3,opt,name=listingCount" json:"listingCount,omitempty"`
+	AvgRating      uint32 `protobuf:"varint,4,opt,name=avgRating" json:"avgRating,omitempty"`
```

Comment:
Q: is it theoretically possible to have negative ratings? (a store/user that sucks so bad)